src/library-authoring/import-course/stepper/ImportStepperPage.tsx

```diff
@@ -40,7 +40,9 @@ export const MigrationStatus = ({
       for (const libraries of Object.values(migrationInfoData)) {
         // The map key in `migrationInfoData` is in camelCase.
         // In the processed map, we use the key in its original form.
-        result[libraries[0].sourceKey] = libraries.map(item => item.targetKey);
+        if (libraries.length !== 0) {
+          result[libraries[0].sourceKey] = libraries.map(item => item.targetKey);
+        }
```

Comment:
This was crashing for me because the backend was returning some courses with an empty library list.

I didn't dig into whether it was an intended change on the backend, since I think this check won't cause any harm here.